cmd/commands/server/factory.go

```diff
@@ -79,16 +79,26 @@ func NewCommandWith(
 			)
 		},
 
-		sessionManagerFactory: func(vpnServerIP string) session.Manager {
-			clientConfigGenerator := openvpn.NewClientConfigGenerator(options.DirectoryRuntime, vpnServerIP, options.Protocol)
+		sessionManagerFactory: func(primitives *tls.Primitives, vpnServerIP string) session.Manager {
```

Comment:
We are creating several instances of `sessionManager`, but it does not state anymore. 
Add TODO for this problem

Reply:
Commented